src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -907,8 +526,6 @@
     object-fit: cover;
     filter: blur(15px);
     opacity: 0.3;
-    /* scale is so there will be no ugly edges because of the blur */
-    transform: scale(1.1);
```

Comment:
I don't know why this was there, but from the comment above it seems like it shouldn't be removed (I don't know what was the original issue)

Reply:
I removed because in the new grid config it keeps letting a space in the bottom because of the picture in the background.